Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -24,5 +24,12 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
 
-    // Add your necessary variables here
+    double amphoursUsed;
+    double previousCurrent;
+    double averageCurrent;
+    double current;
+    bool check1;
```

Comment:
`check1` isn't that good of a name... see if you can call this something more descriptive - what is it trying to check?